src/lib/bump-stats.ts

```diff
@@ -20,7 +20,10 @@ export function bumpAggregatedUniqueStat(
 	getLastBump( group, stat )
 		.then( ( lastBump ) => {
 			if ( lastBump === null ) {
-				return bumpStat( group, stat, bumpInDev );
+				bumpStat( group, stat, bumpInDev );
+				// Track the first occurence of this stat separately
+				bumpStat( `${ group }-first`, stat, bumpInDev );
```

Comment:
This felt like the simplest approach to tracking first-time launches. 

I also experimented with a separate `bumpFirstUniqueStat` function that worked very similarly to `bumpAggregatedUniqueStat`. However, `bumpFirstUniqueStat`  and `bumpAggregatedUniqueStat` did not play nicely together due to the async nature of [`updateLastBump`](https://github.com/Automattic/studio/blob/fed78eb88bb3b91ae37255c94d1a673d681a5e2c/src/lib/bump-stats.ts#L73-L79). Only one would fully execute on the first launch, while the second would be fire a second time when the app was launched again. If we wanted to go with a second `bumpFirstUniqueStat` function, I believe we'd need to refactor the existing functions to use `async/await`, which this comment led me away from doing:

https://github.com/Automattic/studio/blob/fed78eb88bb3b91ae37255c94d1a673d681a5e2c/src/lib/bump-stats.ts#L12-L13

I'd definitely be interested to hear any concerns on the approach I opted for, or other suggestions!

Reply:
Would it make sense to add a comment above each of the `bumpStat( group, stat, bumpInDev );` calls to clearly mark that one is for the first launch, and the second one for subsequent launches in the next interval?

Reply:
Good idea, I went ahead to add comments in https://github.com/Automattic/studio/pull/221/commits/48041a41d442692d4705ead3fd419952b4285d79. There are also comments in the `index.ts` file:

https://github.com/Automattic/studio/blob/48041a41d442692d4705ead3fd419952b4285d79/src/index.ts#L231-L234

Let me know if you think the wording should be tweaked or made clearer in any way!

Reply:
I think it's clear now, thanks for adding those comment.